enzy_htp/structure/structure.py

```diff
@@ -964,6 +965,22 @@ def build_metalcenters(self, dir, ft="PDB"):
 
     #endregion
 
+    def fix_pymol_naming(self) -> None:
+        """Fixes PyMOL's naming of atoms in-place using the PYMOL_TO_ATOM_MAPPER in chemical/residue.py.
+        Args:
+            stru: the Structure that contains the wrong atom names (if any).
+        Returns:
+            Nothing.
+        """
+        for chain in self.chains:
+            for residue in chain.residues:
+                for atom in residue.atoms:
+                    # special case; HA in GLY becomes HA2
+                    if residue.name == "GLY" and atom.name == "HA":
+                        atom.name = "HA2"
+
+                    if atom.name in PYMOL_TO_ATOM_MAPPER:
+                        atom.name = PYMOL_TO_ATOM_MAPPER[atom.name]
 
```

Comment:
This is not a good design of this function. The Structure class should never know anything about PyMol. Every knowledge EnzyHTP knows about PyMol should go to PyMolInterface and pymol_config. 
Also this function here can be decomposed into 2 more modular functions:
1. give the atom mapping that pymol fix needs.
2. batch edit atom names in Structure() given a atom naming mapper as a input. (You can also make sth like this to match the GLY behavior:
{"GLY":{"HA" : "HA2"}, "DEFAULT":default_mapper} and make a getter function for the dict like we did in /chemical
)
The 2nd function can stay in Structure class